tutorials/publish_subscribe/java/build.gradle

```diff
@@ -0,0 +1,49 @@
+plugins {
+    id 'application'
+}
+
+dependencies {
+    implementation files(System.getenv('NDDSHOME') + '/lib/java/nddsjava.jar')
+}
+
+sourceSets {
+    main {
+        java {
+            srcDirs = ['.']
+        }
+    }
+}
+
+task generateCode(type: Exec) {
+    commandLine System.getenv('NDDSHOME') + '/bin/rtiddsgen',
+        '-language', 'Java',
+        'home_automation.idl'
+    onlyIf {
+        !file('DeviceStatus.java').exists()
```

Comment:
TODO: Add the rest of generated files or use a regex, if at the end we follow this approach